.github/workflows/codeql-analysis.yml

```diff
@@ -2,15 +2,10 @@ name: codeql
 
 on:
   push:
-    branches:
-    - v1.10
-    - v1.9
-    - v1.8
-  pull_request:
     branches:
     - master
-  schedule:
-    - cron: "45 6 * * 3"
```

Comment:
https://github.com/cilium/cilium/pull/16241 removed the master run instead of the scheduled run. I'd strongly prefer to keep the scheduled run. It's less frequent and we don't need to run for every single push to master.

Reply:
sounds good i'll put it back. how about pull requests against release branches?

Reply:
Are you sure this setup will work? See current warnings on CodeQL from this PR's Actions: https://github.com/cilium/cilium/actions/runs/861152333
![image](https://user-images.githubusercontent.com/4659919/119152298-7a7b8f00-ba50-11eb-82d9-fd6d754b1746.png)

Reply:
added back push to master trigger 🤷

Reply:
Warning is still here. I don't get it.
https://github.com/cilium/cilium/actions/runs/867731119

Reply:
I think that's just because the workflow on master, doesn't run on master:
https://github.com/cilium/cilium/blob/205f3003c15fc025547346e7688c5b6b04657765/.github/workflows/codeql-analysis.yml#L4-L8

It should be fixed once this is merged. We should really find another way to limit CodeQL runs though, because they take a while :(

Reply:
I'll put my stamp on the current version but agree that it is a poor stopgap.